pages/1.13/monitoring/logging/aggregating/splunk/index.md

```diff
@@ -52,7 +52,7 @@ For each master node in your DC/OS cluster:
         -u dcos-exhibitor.service          \
         -u dcos-gen-resolvconf.service     \
         -u dcos-gen-resolvconf.timer       \
-        -u dcos-history.service            \
+        -u dcos-history.service            \ # deprecated
```

Comment:
I don't think we should add the deprecated comment here, personally. It looks a bit weird on the site. I think if the dcos-history service itself is documented as deprecated, then we can just live this as-is here, and remove it later. This page is about Splunk, not really about the history service.